spec/resolver-fixtures/policy-simple.yml

```diff
@@ -67,4 +67,4 @@ expectation: |
         id: myapp
     role: !group
       account: the-account
-      id: myapp//root-owner
+      id: "/root-owner"
```

Comment:
@kgilpin can you take a look at that? This doesn't look 100% right; OTOH resolving them to `myapp//root-owner` didn't look right either. Is this (non-absolute ids beginning with `/`) something we want to support? I'm thinking this policy should fail at `!group "/root-owner"` at line 4 above.